src/daft-sql/src/planner.rs

```diff
@@ -1066,6 +1058,45 @@ impl<'a> SQLPlanner<'a> {
         }
     }
 
+    /// Plan a `FROM <path>` table factor by rewriting to relevant table-value function.
+    fn plan_relation_path(&self, name: &ObjectName) -> SQLPlannerResult<Relation> {
+        let path = name.to_string();
+        let path = &path[1..path.len() - 1]; // strip single-quotes ' '
```

Comment:
tip: you can also get the unquoted value via `object_name.0[0].value`

Reply:
addressed in: https://github.com/Eventual-Inc/Daft/pull/3683